ocs_ci/helpers/cnv_helpers.py

```diff
@@ -122,7 +122,7 @@ def create_volume_import_source(name=None, url=constants.CNV_CENTOS_SOURCE):
         source_data_obj: The created VolumeImportSource object
 
     """
-    source_data = templating.load_yaml(constants.CNV_VM_STANDALONE_PVC_SOURCE_YAML)
+    source_data = templating.load_yaml(constants.CNV_VM_SOURCE_YAML)
```

Comment:
Because, the naming convention used prior was meant for standalone PVCs. Since our templates are now applicable for DV/DVT as well, going with only standalone pvc convention causes confusion.

Reply:
Okay. lgtm